plugins/inventory/virtualbox.py

```diff
@@ -35,6 +37,17 @@
             description: create vars from virtualbox properties
             type: dictionary
             default: {}
+        enable_advanced_group_parsing:
+            description: >
+                The default group parsing rule (when this setting is set to `false`) is to split the VirtualBox VM's group based on the `/` character and
+                assign the resulting list elements as an Ansible Group.
+                Setting `enable_advanced_group_parsing` to `true` changes this behaviour to match VirtualBox's interpretation of groups according to:
+                https://www.virtualbox.org/manual/UserManual.html#gui-vmgroups
+                Groups are now split using the `,` character, and the `/` character indicates nested groups.
+                So, for a VM that's been configured using `VBoxManage modifyvm "vm01" --groups "/TestGroup/TestGroup2,/TestGroup3"`:
+                - TestGroup2 is a child group of TestGroup
+                - The VM will be part of TestGroup2 and TestGroup3.
```

Comment:
This kind of formatting won't work.
```suggestion
            description:
              - The default group parsing rule (when this setting is set to V(false)) is to split the VirtualBox VM's group based on the V(/) character and
                assign the resulting list elements as an Ansible Group.
              - Setting O(enable_advanced_group_parsing=true) changes this behaviour to match VirtualBox's interpretation of groups according to
                U(https://www.virtualbox.org/manual/UserManual.html#gui-vmgroups).
                Groups are now split using the V(,) character, and the V(/) character indicates nested groups.
              - So, for a VM that's been configured using V(VBoxManage modifyvm "vm01" --groups "/TestGroup/TestGroup2,/TestGroup3"):
                C(TestGroup2) is a child group of C(TestGroup);
                the VM will be part of C(TestGroup2) and C(TestGroup3).
```
(Also check out https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation)

Reply:
Thanks for the link! It's easy to get lost in the various links to all the contributor guides. I'll bookmark this useful page!